### PR TITLE
Add /media route to show brand and logo guidelines

### PR DESCRIPTION
**Pull request by stephendolan**
I added a new page that mimics the lovely https://crystal-lang.org/media/ page

The intention here is that folks are going to use our brand/colors/logo anyways as the framework grows in popularity, so we might as well make it easy and set clear boundaries for doing so.

I basically copied and pasted Crystal's guidelines. If we want to relax/remove any of these or add additional ones, I'm totally open to that (I already broke one with the LuckyCasts logo 😆 ).

Here's what the page looks like:
CleanShot 2021-07-09 at 10 25 04
CleanShot 2021-07-09 at 10 25 32

It's all styled to work on mobile as well, but I won't add screenshots to the PR unless y'all would like them.

### CODE REVIEW

**Comment by stephendolan** [comment]
Oh, and those two Logo/Icon images will download a PNG when clicked. Not super obvious until you see the hover effect:
CleanShot 2021-07-09 at 10 28 09

File: src/pages/media/index_page.cr [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        render_allowed_list_item("Use them to advertise something you’ve created with Lucky.")
+        render_allowed_list_item("Use them in blog posts, articles or news about Lucky.")
+        render_allowed_list_item("Use them for educational purposes, as long as it’s available for free.")
+      end
+
+      h2 "Don’t do these things with the Lucky logos and brand:", class: "mt-8 mb-4 text-xl font-semibold"
+      ul class: "space-y-2" do
+        render_disallowed_list_item("Change their colors, dimensions or add your own text.")
+        render_disallowed_list_item("Sell Lucky artwork or merchandise without permission.")
+        render_disallowed_list_item("Integrate your own logo with Lucky logos.")
```
Comment (jwoertink) [inline src/pages/media/index_page.cr:50]: Does this mean the LuckyCasts logo is in violation? 🤔  😛

**Review by jwoertink — APPROVED** [review]
I think this is awesome.

File: src/pages/media/index_page.cr (hunk 1 above)
Comment (stephendolan) [reply src/pages/media/index_page.cr:50]: Haha yeah, that's what I was referencing in the description 😬 

Maybe I need to add "(except LuckyCasts pretty pls, see https://github.com/luckyframework/lucky/issues/23)" 😛

Comment (paulcsmith) [reply src/pages/media/index_page.cr:50]: Maybe “without permission” or add to the “contact us” part

File: src/pages/media/index_page.cr [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        render_disallowed_list_item("Use the Lucky logos as your brand, company or app image.")
+        render_disallowed_list_item("Use the logo in a way that could suggest sponsorship or endorsement by Lucky.")
+      end
+
+      h2 "Don’t hesitate to contact us if:", class: "mt-8 mb-4 text-xl font-semibold"
+      ul class: "space-y-2" do
+        render_contact_list_item("You want permission to sell Lucky merchandise.")
+        render_contact_list_item("You want to use these logos for commercial purposes.")
+        render_contact_list_item("You have questions about the use of Lucky brand and logos.")
```
Comment (paulcsmith) [inline src/pages/media/index_page.cr:59]: I love how clean this is with the private methods. HTML in crystal is great!

**Review by paulcsmith — APPROVED** [review]
Looks good!

**Pull request merged**